lib/dl_connector_bundle_chs3/dl_connector_bundle_chs3/chs3_gsheets/core/constants.py

```diff
@@ -4,9 +4,14 @@
     NotificationType,
 )
 
+from dl_connector_bundle_chs3.constants import (
+    NOTIF_TYPE_DATA_UPDATE_FAILURE,
+    NOTIF_TYPE_STALE_DATA,
+)
+
 
 CONNECTION_TYPE_GSHEETS_V2 = ConnectionType.declare("gsheets_v2")
 SOURCE_TYPE_GSHEETS_V2 = DataSourceType.declare("GSHEETS_V2")
 
-NOTIF_TYPE_GSHEETS_V2_STALE_DATA = NotificationType.declare("stale_data")
-NOTIF_TYPE_GSHEETS_V2_DATA_UPDATE_FAILURE = NotificationType.declare("data_update_failure")
+NOTIF_TYPE_GSHEETS_V2_STALE_DATA = NOTIF_TYPE_STALE_DATA
+NOTIF_TYPE_GSHEETS_V2_DATA_UPDATE_FAILURE = NOTIF_TYPE_DATA_UPDATE_FAILURE
```

Comment:
If these notifications are basically the same, I think there is no need to specify their copies with specific names. What I mean is, we can just use NOTIF_TYPE_STALE_DATA & NOTIF_TYPE_DATA_UPDATE_FAILURE where they are needed, without these assignments